crypto/params.c

```diff
@@ -15,7 +15,7 @@
     if (p->return_size != NULL) \
         *p->return_size = (sz)
 
-static const OSSL_PARAM *param_locate(const OSSL_PARAM *p, const char *key)
+OSSL_PARAM *OSSL_PARAM_locate(OSSL_PARAM *p, const char *key)
```

Comment:
Is there a reason why you removed `const`?

Reply:
There was when I'd misunderstood how _PTR worked.  I'll put it back.